alt_evaluator.py

```diff
@@ -7,34 +7,27 @@
 
 
 class AltEvaluator(object):
-    def __init__(self, path_to_lmdb_dir):
+    def __init__(self, path_to_lmdb_dir, number_images_to_evaluate):
         transform = transforms.Compose([
             transforms.CenterCrop([54, 54]),
             transforms.ToTensor(),
             transforms.Normalize([0.5, 0.5, 0.5], [0.5, 0.5, 0.5])
         ])
-        self.batch_size = 128
-        self._loader = torch.utils.data.DataLoader(Dataset(path_to_lmdb_dir, transform), batch_size=self.batch_size, shuffle=False)
+        self.dataset = Dataset(path_to_lmdb_dir, transform)
+        if number_images_to_evaluate:
+            self.dataset = self.dataset[0:int(number_images_to_evaluate)]
+        self._loader = torch.utils.data.DataLoader(self.dataset, batch_size=32, shuffle=False)
```

Comment:
Use `self.batch_size` here so we can dynamically change the batch size if we need